src/Chainweb/Version.hs

```diff
@@ -521,20 +519,20 @@ newtype BlockRate = BlockRate Seconds
 -- number of seconds we expect to pass while a miner mines on various chains,
 -- eventually succeeding on one.
 --
-blockRate :: ChainwebVersion -> Maybe BlockRate
-blockRate Test{} = Nothing
-blockRate TimedConsensus{} = Just $ BlockRate 4
-blockRate PowConsensus{} = Just $ BlockRate 10
-blockRate TimedCPM{} = Just $ BlockRate 4
+blockRate :: ChainwebVersion -> BlockRate
+blockRate Test{} = BlockRate 0
```

Comment:
What will happen if adjustment is called for this? Before there would be a meaningful runtime error:

```haskell
error $ "No BlockRate available for given ChainwebVersion: " <> show v
```

With this change will the code just crash with division by 0 or similar, will it provide a meaningful error, or will it just work?

Reply:
Unknown, but the `Test` value is rarely used.

Reply:
Maybe just replace it by something like `error "ChainwebVersion 'Test' doesn't most not use the BlockRate setting".

That would help identify the error if a tests hits it. Otherwise, we my just get `Division by zero` but have no idea where it's coming from.